sda-commons-server-spring-data-mongo/src/main/java/org/sdase/commons/server/spring/data/mongo/annotation/DocumentType.java

```diff
@@ -0,0 +1,13 @@
+package org.sdase.commons.server.spring.data.mongo.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+public @interface DocumentType {
```

Comment:
To me, `DocumentType` does not convey the expected semantics. Maybe something like `SubType`?

Reply:
Spring data also knows about a `@TypeAlias` annotation. Could we just use that? Maybe we don't even have to care about the stored `_class` if we suggest and document to case `@TypeAlias`? I will play a little bit with that and try out.